api

```diff
@@ -675,6 +675,73 @@ debian_ppa_installer() { #setup a PPA on a Debian distro. Arguments: ppa_name di
     apt_update || exit 1
   fi
 }
+
+terminal_manage() { # wrapper for the original terminal_manage function to terminal_mange_multi
+  #function to install/uninstall/update/refresh one app (or filelist) - uses a terminal and refreshes the app list
+  if [ "$1" == 'uninstall' ];then
+    action=uninstall
+  elif [ "$1" == 'install' ];then
+    action=install
+  elif [ "$1" == 'update' ];then
+    action=update
+  elif [ "$1" == 'refresh' ];then
+    action=refresh
+  else
+    error "terminal_manage(): Must specify an action: either 'install' or 'uninstall' or 'update' or 'refresh'"
+  fi
+  
+  app="$2" #one app name (or filelist of the format 'filelist:path/to/file:path/to/another/file')
+  terminal_manage_multi "$action $app"
+}
+
+terminal_manage_multi() { #function to install/uninstall/update/refresh multiple apps (or filelists) - uses a terminal and refreshes the app list
+  queue="$1" #one or multiple actions and app names (or filelists of the format 'filelist:path/to/file:path/to/another/file')
```

Comment:
also just to note, I did not bother to add any complicated checks on the contents of the `queue` here since the `"${DIRECTORY}/manage" daemon "$queue"` already validates the contents itself. doing so here would be redundant code.